Protocaster/AppDelegate.swift

```diff
@@ -32,9 +32,22 @@ class AppDelegate: NSObject, NSApplicationDelegate, DTBonjourDataConnectionDeleg
 			}
 		}
 	}
+    
+    private final func sceneURL(fromURL URL: NSURL) -> NSURL? {
+        var isDirectoryValue: AnyObject?
+        URL.getResourceValue(&isDirectoryValue, forKey: NSURLIsDirectoryKey, error: nil)
+        
+        let isDirectory = (isDirectoryValue as? NSNumber)?.boolValue ?? false
+        
+        if !isDirectory {
+            return URL.URLByDeletingLastPathComponent
+        }
+        
+        return URL
+    }
 
 	private func selectedPathDidChange(newURL: NSURL?) {
-		if let URL = newURL {
+		if let URL = (newURL >>- sceneURL) {
```

Comment:
Can we start phasing out usage of `bind` in favor of e.g. Swift 1.2's multiple destructuring if-let?